src/Components/ShareProfile.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import GetIcons from './Icons/GetIcons'
+import PropTypes from 'prop-types'
+import './ShareProfile.css'
+import ShareIcon from './ShareIcon'
+
+export default function ShareProfile({ username }) {
+  const CopyLink = () => {
+    navigator.clipboard.writeText(`http://linkfree.eddiehub.org/${username}`)
+    alert('Link copied to clipboard.')
```

Comment:
Second suggestion 😄 : Maybe we can use a [PrimeReact-Toast](https://www.primefaces.org/primereact/toast/) instead of a plain alert

Reply:
for sure! I have never tried Prime react 🙄 . Thanks for telling me about this :D

Reply:
> Maybe it’s something we need to make clearer in the contributing readme

Yeah 🤔

Reply:
@Cahllagerfeld changed it to prime toast :)

Reply:
> > for sure! I have never tried Prime react 🙄 . Thanks for telling me about this :D
> 
> Maybe it’s something we need to make clearer in the contributing readme

Do you want create an issue for it?